lang/python/python-package.mk

```diff
@@ -10,6 +10,10 @@
 python_mk_path:=$(dir $(lastword $(MAKEFILE_LIST)))
 include $(python_mk_path)python-host.mk
 
+ifeq ($(findstring python,$(PKG_BUILD_DEPENDS)),)
```

Comment:
I always though that adding `PKG_BUILD_DEPENDS:=python` would also include `python/host`; though I admit I never did the digging to make sure.
Could you provide some links to `random CircleCI failures` ?

I can take a look at these failures and maybe we can have a better idea of what's going on.

Reply:
No offhand but @neheb complains about them all the time...perhaps he has some recent examples.  I do know it failed for me without the build depends, but I don't remember if I did it with just python or python/host.  DEPENDS:= however is not enough to get it to work, need a PKG_BUILD_DEPENDS...I'll get some logs.

Reply:
let's remove commit ` python: python3: Add missing python{3}/host PKG_BUILD_DEPENDS ` from this PR until we settle the random failures ; logs indeed help